content/docs/installation.md

```diff
@@ -89,6 +89,10 @@ npm install --save react react-dom
 
 Both Yarn and npm download packages from the [npm registry](http://npmjs.com/).
 
+>  Note:
+>
+>  To prevent potential incompatibilities, all `react` packages should use the same version (eg `react@16.2.0`, `react-dom@16.2.0`, `react-test-renderer@16.2.0`).
```

Comment:
Please "e.g.", not "eg". Even better, "for example" or "such as".

I also don't think we should hardcode `16.2.0` there, people will just use that. Any other ideas how we can communicate this?

Reply:
I suggest:
+ e.g. => such as
+ remove version of package

**It is going to be**
To prevent potential incompatibilities, all `react` packages should use the same version (such as `react`, `react-dom`, `react-test-renderer`, ...).

Reply:
what do you think about it?
@gaearon

Reply:
> I also don't think we should hardcode 16.2.0 there, people will just use that. Any other ideas how we can communicate this?

Hm. Really? I guess that's possible, but I think the "eg" or "for example" prefix should make that clear.

Reply:
>Hm. Really?

Maybe I just have very low expectations for people reading the docs. :-) But if there's a way to phrase something that's more easily copy-pasteable by someone who barely knows any English, it's usually better.

Reply:
Yeah. Just seems like everything else that comes to mind might be confusing in different ways. Maybe just?

> To prevent potential incompatibilities, all `react` packages should use the same version. (This includes `react`, `react-dom`, `react-test-renderer`, etc.)

Reply:
Sounds good to me

Reply:
I have just updated with the words from @bvaughn